model/src/main/kotlin/Project.kt

```diff
@@ -85,6 +85,11 @@ data class Project(
      */
     val vcsProcessed: VcsInfo = vcs.normalize(),
 
+    /**
+     * The description of project.
```

Comment:
Do you guys believe empty values should be serialized or omit from serialization?
(Former requires updating all expected results)

Reply:
I believe empty / default values should be omitted (for smaller file size).